### ISSUE TITLE
[STALL] Can't run manylinux1 container on "new" Debian

### ISSUE THREAD

**Issue by Mizux** [google/or-tools#853]
On recent debian, it is no possible anymore to run manylinux images, thus we can't generate `ortools` Python packages on Google Corp machine...

```shell
docker run --rm -it quay.io/pypa/manylinux1_x86_64

echo $?                                           
139
```

related to https://github.com/pypa/manylinux/issues/141

> This affects any docker image that uses glibc 2.13 or earlier. This
includes both CentOS 5 and CentOS 6.

e.d. manylinux image is based on `CentOS 5`...
src: https://github.com/pypa/manylinux/blob/master/docker/Dockerfile-x86_64#L1

WORKAROUND: Build docker images on MacOS which is (still) not impacted by this kernel bit flag.

**Comment by Mizux** [comment]
> reboot and add the option 'vsyscall=emulate' to
the kernel command line

`PEP571` which follows `PEP513` talk about it...
src: https://www.python.org/dev/peps/pep-0571/#compatibility-with-kernels-that-lack-vsyscall
TLDR:
> building manylinux2010 image itself, still requires a host kernel that provides the vsyscall mechanism, but the resulting image can be both run on hosts that provide it and those that do not. 

note: currently quay.io/pypa doesn't provide manylinux2010
cf https://github.com/pypa/manylinux/issues/179
